rancher/k3s
# Use rancher-mirrored busybox for local-path-provisioner

#### Proposed Changes ####

* Override local-storage (local-path-provisioner) helper image to point at rancher mirrored multiarch busybox image.
* Add busybox to airgap image list

#### Types of Changes ####

* Chart update
* Airgap

#### Verification ####

1. Use local-storage on an airgap system
1. Notice that it doesn't fail due to missing busybox image

#### Linked Issues ####

Related to #1908

#### Further Comments ####

Will be fixed upstream by
https://github.com/rancher/local-path-provisioner/pull/135/ but we're
not going to update the LPP image right now since it's undergoing some
changes that we don't want to pick up at the moment.
